packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts

```diff
@@ -519,12 +523,14 @@ export class AmplifyAuthTransform extends AmplifyCategoryTransform {
         );
       }
       if (Array.isArray(value)) {
-        this._authTemplateObj.addCfnParameter(
-          {
-            type: 'CommaDelimitedList',
-          },
-          `${key}`,
-        );
+        if (key !== 'userAutoVerifiedAttributeUpdateSettings') {
+          this._authTemplateObj.addCfnParameter(
+            {
+              type: 'CommaDelimitedList',
+            },
+            `${key}`,
```

Comment:
Assuming `key` is a string:
```suggestion
            key,
```